netbox/dcim/models/__init__.py

```diff
@@ -2191,19 +2191,20 @@ def clean(self):
                 f"Incompatible termination types: {self.termination_a_type} and {self.termination_b_type}"
             )
 
-        # A RearPort with multiple positions must be connected to a RearPort with an equal number of positions
+        # Check that a RearPort isn't connected to something silly
```

Comment:
What is "something silly?" Be explicit.

Reply:
Sorry, should have added to TODO marker for myself there :)
Will fix!